packages/blade/src/components/Table/TableBody.web.tsx

```diff
@@ -162,10 +162,10 @@ export const StyledCell = styled(Cell)<{
 }));
 
 export const CellWrapper = styled(BaseBox)<{
-  rowDensity: NonNullable<TableProps<unknown>['rowDensity']>;
+  $rowDensity: NonNullable<TableProps<unknown>['rowDensity']>;
```

Comment:
Also fixed this issue, where rowDensity is being passed as HTML prop and react was complaining since it's not a valid html attribute

Reply:
Can you add a comment for the same?

Reply:
Doesn't need a comment, we are doing this in multiple places already just that we missed this one. 

And it's a standard prop naming feature present in styled components. https://styled-components.com/docs/api#transient-props